### PR TITLE
make openglcontrol pointer event work

### PR DESCRIPTION
**Pull request by Coloryr**
## What does the pull request do?
make openglcontrol pointer event work

## What is the current behavior?
Add render to openglcontrol

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/avalonia-docs with user documentation

## Fixed issues
#10912 

## Dicussions
https://github.com/AvaloniaUI/Avalonia/discussions/10879

### CODE REVIEW

**Comment by Coloryr** [comment]
If control not have `DrawList` the `HitTest` allways false

**Comment by maxkatz6** [comment]
@Coloryr I think implementing ICustomHitTest on OpenGlControl instead would have less overhead. But I am not 100% sure if it's the best solution either.

**Comment by Coloryr** [comment]
> @Coloryr I think implementing ICustomHitTest on OpenGlControl instead would have less overhead. But I am not 100% sure if it's the best solution either.

It may be better to use ICustomHitTest, i found that some controls that do not support this event are missing DrawList

File: src/Avalonia.OpenGL/Controls/OpenGlControlBase.cs [hunk 1]
```diff
@@ -263,29 +258,9 @@ protected virtual void OnOpenGlLost()
         
         protected abstract void OnOpenGlRender(GlInterface gl, int fb);
 
-        private class OpenGLCustomDrawOperation : ICustomDrawOperation
+        public bool HitTest(Point point)
```
Comment (maxkatz6) [inline src/Avalonia.OpenGL/Controls/OpenGlControlBase.cs:261]:
```suggestion
        bool ICustomHitTest.HitTest(Point point)
```

Just to remove this method from code auto-completion.

**Pull request closed without merging**